src/css/index.scss

```diff
@@ -0,0 +1,134 @@
+//public
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0 auto;
+  padding: 0;
+  border: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji',
+    'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+}
+a {
+  text-decoration: none;
+}
+h2 {
+  margin: 0 auto;
+}
+input,
+textarea,
+button {
+  background: none;
+  outline: none;
+  border: 0;
+  resize: none;
+}
+input:focus,
+textarea:focus,
+button {
+  border: none;
+}
+
+//informaion sections
+//quick id section
+.banner_title {
+  color: #fff;
+  letter-spacing: 3.8px;
+  font-size: 36px;
+  font-weight: 600;
+}
+.banner_title:after {
+  content: '';
+  display: block;
+  width: 36px;
+  margin-top: 20px;
+  border: 4px solid rgb(233, 49, 67);
+}
+#banner_information {
```

Comment:
Just a suggestion to use class rather than ID.
Class is easier to override and can be re-used if needed.